proto/dwtools/abase/l1/StrBasic.s

```diff
@@ -4990,11 +4980,21 @@ Dmytro : covered
  * // returns o.splits = [ 'a\n', 'b', 'c' ];
  * // returns o.spans = [ 1, 3, 4, 5 ];
  *
- * @method strLinesNearest
+ * @param { Object } o - Options.
+ * @param { String } [ o.src ] - Source string.
+ * @param { Array|Number } [ o.range ] - Sets range of lines to select from( o.src ) or single line number.
+ * @param { Number } [ o.numberOfLines ] - Sets number of lines to select.
+ * @returns { Object } o - Returns object with Options with fields:
+ * @returns { Array } [ o.splits ] - Array with three entries:
+ * o.splits[ 0 ] and o.splits[ 2 ] contains a string with the nearest lines,
+ * and o.splits[ 1 ] contains the substring corresponding to the range.
+ * @returns { Array } [ o.spans ] - Array with indexes of begin and end of nearest lines.
+ *
  * @throws { Exception } Throw an exception if no argument provided.
  * @throws { Exception } Throw an exception if( o.src ) is not a String.
  * @throws { Exception } Throw an exception if( o.charsRangeLeft ) is not a Array or Number.
  * @throws { Exception } Throw an exception if( o ) is extended by unknown property.
+ * @method strLinesNearest
```

Comment:
In this namespace use keywort `function` instead of `method`. This namespace is not a class.